playbooks/shellcheck.yaml

```diff
@@ -5,7 +5,8 @@
       become: yes
       package:
         name:
-          - golang-github-cpuguy83-go-md2man
```

Comment:
It would be nicer to split the changes related to switching `playbooks/shellcheck.yaml` to Fedora 32 into a separate commit. For a moment, I was thinking that this change from `golang-github-cpuguy83-go-md2man` to `golang-github-cpuguy83-md2man` is related to some sort of typo.

Reply:
Ok, will do.

Reply:
I took the liberty to split them out and merge them via https://github.com/containers/toolbox/pull/441 and https://github.com/containers/toolbox/pull/442